meta-zephyr-sdk/recipes-devtools/xilinx_qemu/xilinx-qemu_git.bb

```diff
@@ -0,0 +1,211 @@
+#  --enable-spice           enable spice
+#  --enable-rbd             enable building the rados block device (rbd)
+#  --disable-libiscsi       disable iscsi support
+#  --enable-libiscsi        enable iscsi support
+#  --disable-libnfs         disable nfs support
+#  --enable-libnfs          enable nfs support
+#  --disable-smartcard-nss  disable smartcard nss support
+#  --enable-smartcard-nss   enable smartcard nss support
+#  --disable-libusb         disable libusb (for usb passthrough)
+#  --enable-libusb          enable libusb (for usb passthrough)
+#  --disable-usb-redir      disable usb network redirection support
+#  --enable-usb-redir       enable usb network redirection support
+#  --enable-lzo             enable the support of lzo compression library
+#  --enable-snappy          enable the support of snappy compression library
+#  --disable-guest-agent    disable building of the QEMU Guest Agent
+#  --enable-guest-agent     enable building of the QEMU Guest Agent
+#  --with-vss-sdk=SDK-path  enable Windows VSS support in QEMU Guest Agent
+#  --with-win-sdk=SDK-path  path to Windows Platform SDK (to build VSS .tlb)
+#  --disable-seccomp        disable seccomp support
+#  --enable-seccomp         enables seccomp support
+#  --with-coroutine=BACKEND coroutine backend. Supported options:
+#                           gthread, ucontext, sigaltstack, windows
+#  --disable-coroutine-pool disable coroutine freelist (worse performance)
+#  --enable-coroutine-pool  enable coroutine freelist (better performance)
+#  --enable-glusterfs       enable GlusterFS backend
+#  --disable-glusterfs      disable GlusterFS backend
+#  --enable-gcov            enable test coverage analysis with gcov
+#  --gcov=GCOV              use specified gcov [$gcov_tool]
+#  --disable-tpm            disable TPM support
+#  --enable-tpm             enable TPM support
+#  --disable-libssh2        disable ssh block device support
+#  --enable-libssh2         enable ssh block device support
+#  --disable-vhdx           disables support for the Microsoft VHDX image format
+#  --enable-vhdx            enable support for the Microsoft VHDX image format
+#  --disable-quorum         disable quorum block filter support
+#  --enable-quorum          enable quorum block filter support
+#  --disable-numa           disable libnuma support
+#  --enable-numa            enable libnuma support
+
+#--disable-fdt: Required for arm-generic-fdt machine type
+
+QEMUS_BUILT = "aarch64-softmmu"
+QEMU_FLAGS = "--disable-docs  --disable-sdl --disable-debug-info  --disable-cap-ng \
+  --disable-libnfs --disable-libusb --disable-libiscsi --disable-usb-redir --disable-linux-aio\
+  --disable-guest-agent --disable-vnc-png  --disable-seccomp \
+  --disable-tpm  --disable-numa --disable-glusterfs \
+  --disable-virtfs --disable-xen --disable-curl --disable-attr --disable-curses \
+  "
+
+do_configure() {
+    ${S}/configure ${QEMU_FLAGS} --target-list="${QEMUS_BUILT}" --prefix=${xilinx_qemu_prefix}
+}
```

Comment:
Noticing diff between standard recipe doing:

`--sysconfdir=${sysconfdir} --libexecdir=${libexecdir} --localstatedir=${localstatedir}`

And thoughts on matching it?

Reply:
If we do that, we will be basically overwriting the shared exes and libs (the conflict issue described above).

Once `--prefix` is set to `/usr/xilinx`, the QEMU configure automatically figures out adequate `--sysconfdir, --libexecdir, --localstatedir` under `/usr/xilinx`. This effectively prevents any potential conflict with the upstream QEMU that resides in `/usr`.

Reply:
Was guessing that was the case, do you mind adding a comment into the recipe stating that we differ because of this reason.

Reply:
```
$ ls -l /opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx -xR
/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx:
bin  libexec  share

/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx/bin:
ivshmem-client  ivshmem-server  qemu-img  qemu-io  qemu-nbd  qemu-pr-helper  qemu-system-aarch64

/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx/libexec:
qemu-bridge-helper

/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx/share:
qemu

/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx/share/qemu:
acpi-dsdt.aml       bamboo.dtb        bios-256k.bin          bios.bin             efi-e1000e.rom            efi-e1000.rom
efi-eepro100.rom    efi-ne2k_pci.rom  efi-pcnet.rom          efi-rtl8139.rom      efi-virtio.rom            efi-vmxnet3.rom
keymaps             kvmvapic.bin      linuxboot.bin          linuxboot_dma.bin    multiboot.bin             openbios-ppc
openbios-sparc32    openbios-sparc64  palcode-clipper        petalogix-ml605.dtb  petalogix-s3adsp1800.dtb  ppc_rom.bin
pxe-e1000.rom       pxe-eepro100.rom  pxe-ne2k_pci.rom       pxe-pcnet.rom        pxe-rtl8139.rom           pxe-virtio.rom
QEMU,cgthree.bin    qemu-icon.bmp     qemu_logo_no_text.svg  QEMU,tcx.bin         qemu_vga.ndrv             s390-ccw.img
s390-netboot.img    sgabios.bin       skiboot.lid            slof.bin             spapr-rtas.bin            trace-events-all
u-boot.e500         vgabios.bin       vgabios-cirrus.bin     vgabios-qxl.bin      vgabios-stdvga.bin        vgabios-virtio.bin       vgabios-vmware.bin

/opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/xilinx/share/qemu/keymaps:
ar         bepo  common  cz  da  de  de-ch  en-gb  en-us  es  et  fi  fo  fr  fr-be  fr-ca  fr-ch  hr  hu  is  it  ja  lt  lv  mk    modifiers  nl    nl-be   no  pl  pt  pt-br  ru     sl     sv  th  tr
```

Reply:
Ok, I will add comments regarding these two issues in the recipe.